### PR TITLE
Schema export: export bare types as non-nullable

### PR DESCRIPTION
**Pull request by moon-chilled**
Aside: does it make sense to have both `GQLDNullable` and `GQLDNonNullable`?  Would it make more sense for one of them be removed?

### CODE REVIEW

**Comment by moon-chilled** [comment]
Or maybe another option: make nullability be a flag on existing types.

**Comment by burner** [comment]
I'll find tomorrow for this. Sorry for the delay

File: source/graphql/schema/toschemafile.d [hunk 1]
```diff
@@ -158,15 +158,15 @@ Visibility traceType(GQLDType t, ref TraceableType[string] tab) {
 	return tab[map.name].vis;
 }
 
-string gqldTypeToString(const(GQLDType) t, string nameSuffix = "") {
+string gqldTypeToString(const(GQLDType) t, string nameSuffix = "", bool nonNullable = true) {
```
Comment (burner) [inline source/graphql/schema/toschemafile.d:161]: could you please turn the bool into a std.typecons.Flag!"NonNullable" passing boolean around is not nice.

I properly do it somewhere as well but, consider this a "do as I say, not as I do" situtation.

**Review by burner — CHANGES_REQUESTED** [review]
Could you please a a Nullable!(bare type) test just to make sure that is handled correctly.

I did see that in the diff.

**Comment by moon-chilled** [comment]
@burner ping

**Comment by burner** [comment]
sorry for taking so long

**Pull request merged**